resource-managers/yarn/src/test/resources/META-INF/services/org.apache.spark.deploy.yarn.security.ServiceCredentialProvider

```diff
@@ -1 +0,0 @@
-org.apache.spark.deploy.yarn.security.TestCredentialProvider
```

Comment:
`ConfigurableCredentialManagerSuite`. It is loaded by `ServiceLoader` implicitly.